content/en/docs/contribute/style/style-guide.md

```diff
@@ -49,17 +49,18 @@ When you refer specifically to interacting with an API object, use [UpperCamelCa
 Don't split the API object name into separate words. For example, use
 PodTemplateList, not Pod Template List.
 
-Refer to API objects without saying "object," unless omitting "object"
-leads to an awkward construction.
+There is a subtle distinction between formal API objects and general concepts. Use "object" where appropriate to clarify the meaning of terms. Calling out a "Pod object" may be appropriate when moving from general concepts to specific API objects. For example, "Pods may have multiple volume mounts. The Pod object includes a Volumes field."
+
+Do not use "object" repetitively, such as where it's clear a term is an API object. 
```

Comment:
Hello @geoffcline , Some feedback:
Good idea to relax the use of "object" with resource types. If in doubt, I think it is a good idea
to log an issue to propose a change (not everyone is able to attend a meeting).
Note: The current guidance was probably added to curb overuse.

nit: I am tempted to drop line 54.
Does this suggest that the content should introduce an API term with "object"
and then drop the use of "object" with further instances of the term within a page?
I don't know whether the guide needs to go into this level of detail on the usage of "object".
What do you think about adding:
`You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence.
Note that Kubernetes resource names are capitalized. However, capitalization alone is not a reliable
way to identify Kubernetes resource types in a sentence`.

Regarding the statement about "weak signal and lost due to ...", this seems to be referring to 
the same possible issue of using code formatting for resource types?
I can't tell whether this type of visual hint to the reader is helpful. What do you think?

Reply:
I like your wording on clarifying resource types, thanks. 

I agree about the similarities between capitalization and code style. I do think code style sends a bit clearer signal, and is a better way to indicate something is a string used in code (and thus shouldn't be translated). 

I lean towards offering people more options, but I understand we may want the style guidelines to be more definitive.

I'm optimistic my next draft will address these questions.